common_vars.sh

```diff
@@ -94,6 +94,15 @@ function get_blas {
     fi
 }
 
+function get_freetype {
+    yum install -y libpng-devel libjpeg-devel
```

Comment:
I guess; I just need them to get matplotlib and pillow built. If matplotlib
itself was the target, I may have tried downloading a later source package.